src/prog_models/prognostics_model.py

```diff
@@ -1272,13 +1278,19 @@ def optimization_fcn(params):
         
         params = np.array([self.parameters[key] for key in keys])
 
-        res = minimize(optimization_fcn, params, method=method, bounds = config['bounds'], options=config['options'])
+        res = minimize(optimization_fcn, params, method=method, bounds = config['bounds'], options=config['options'], tol=tol)
```

Comment:
```suggestion
        res = minimize(optimization_fcn, params, method=method, bounds=config['bounds'], options=config['options'], tol=tol)
```
Minor change- consistent spacing around '='